### DISCUSSION TITLE
PictureThumbnailURL is null for Excel Files not found for Word

### DISCUSSION THREAD

**Discussion by londify** [microsoft-search/pnp-modern-search#3419] [Q&A]
Hi,

I am having difficulty getting the PictureThumbnailURL managed property to bring back data in the search results web part for Excel files and Word Files

For excel files the PictureThumbnailURL  will comeback as null when looking at the debug view in the PnP Search Results web part but if I do a search API rest call in the browser there is a value for the PictureThumbnailURL.

this is returned via search /_api/v2.1/drives/b!CPSH3jmrREGRBAdh3IJofeWBcDvU0OBMslNWrARFy2rMqABasq04SrQlIpj5DMIs/items/01O2LCROM3OUKR2FCG5BCIPFOQ2ZVVTQEB/thumbnails/0/c400x99999/content?prefer=noRedirect</d:Value>

and null in the web part

For word files the PictureThumbnailURL returns a value but it does not render an actual image.  When I paste the URL from the managed property in the browser I get a file not found:
{
    "error": {
        "code": "accessDenied",
        "innerError": {
            "code": "sharesAccessDenied"
        },
        "message": "The system cannot find the file specified. (Exception from HRESULT: 0x80070002)"
    }
}
This happens for both the Search API and the Managed Property

search REST API for docx
/_api/v2.1/shares/u!aHR0cHM6Ly9jaXR5b2Zsb25kb24uc2hhcmVwb2ludC5jb20vc2l0ZXMvdGVhbS1sb25kb24vX2xheW91dHMvMTUvRG9jLmFzcHg/driveItem/thumbnails/0/c400x99999/content?prefer=noRedirect</d:Value>

managed property for docx
/_api/v2.1/shares/u!aHR0cHM6Ly9jaXR5b2Zsb25kb24uc2hhcmVwb2ludC5jb20vc2l0ZXMvdGVhbS1sb25kb24vX2xheW91dHMvMTUvRG9jLmFzcHg/driveItem/thumbnails/0/c400x99999/content?prefer=noRedirect

**Comment by kasperbolarsen** [comment]
Which version of PnP modern search? 

Try using SP Editor or SharePoint search query tool and check the value

**Reply by londify** [reply]
Hi,

We are using 4.9.3 and when using the SharePoint search query tool 2.10 I do get a result for the excel that works and a result for word that does not render.  The same behavior while using the api and PnP
